src/main/java/com/exasol/adapter/sql/SqlPredicateOr.java

```diff
@@ -1,7 +1,6 @@
 package com.exasol.adapter.sql;
 
 import com.exasol.adapter.AdapterException;
-import com.google.common.base.Joiner;
```

Comment:
Check if this allows us to get rid of a dependency. That would be great!

Reply:
I'll do it the next commit, because some Joiners are still in the code and I need to get rid of them.